crates/bevy_pbr/src/render/mesh_view_types.wgsl

```diff
@@ -85,3 +85,12 @@ struct ClusterOffsetsAndCounts {
     data: array<vec4<u32>>,
 };
 #endif
+
+struct Globals {
+    // The time since startup in seconds
```

Comment:
When wrapping, this comment should clarify that it's the time since startup modulo something.